### PR TITLE
Tests+pathfinder update+ wi-fi fix

### PR DESCRIPTION
**Pull request by Anthony1911**
Added modified robot movement tests according to current state of the system

### CODE REVIEW

**Comment by Anthony1911** [comment]
I also updated PathFinder according to class diagram now

File: ROVU/Core/src/controller/DataObject.java [hunk 1]
```diff
@@ -1,4 +1,23 @@
 package controller;
```
Comment (elizaII) [inline ROVU/Core/src/controller/DataObject.java]: I think this should be a part of the model, what do you guys think? @Anthony1911 @nicket97

**Comment by elizaII** [comment]
`robotMoveTest_1` and `robotMoveTest_2` failed for me 😕

File: ROVU/Core/src/controller/DataObject.java (hunk 1 above)
Comment (Anthony1911) [reply ROVU/Core/src/controller/DataObject.java]: Not really, it can either be part of Robot or Controller, as its used only in those two areas.

Comment (elizaII) [reply ROVU/Core/src/controller/DataObject.java]: Then I'd say a part of the Robot

Comment (Anthony1911) [reply ROVU/Core/src/controller/DataObject.java]: Agree

**Pull request merged**